windows-10-setup.md

```diff
@@ -7,45 +10,132 @@
-    - `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
-    - Restart the terminal and open an Ubuntu tab
+    - Enter this command into your terminal (note that it's one long line, even if it displays as two lines on the page where you're reading this): `sh -c "$(curl -fsSL https://raw.githubusercontent.com/ohmyzsh/ohmyzsh/master/tools/install.sh)"`
+    - Restart the terminal and open an Ubuntu tab with the little plus button in the top left corner
+    - If the prompt in your terminal window is now a little arrow and a tilde (~), instead of "yourname@...", that's OK (you'll change it again in a later step)
 1. Install VS Code if it isn't already installed
+    - https://code.visualstudio.com/download
 1. In your Ubuntu terminal, open VS Code with `code .`
 1. Install the following VS Code extensions
     - ESLint
-    - WSL - Remote
+    - Remote - WSL
     - Live Share (online students only)
     - vscode-icons (optional, but pretty :wink:)
     - Bracket Pair Colorizer (optional)
     - GitLens (optional)
 1. Restart your terminal
 1. Open VS Code again with `code .`
     - This should begin downloading the VS Code Server
-    - When prompted for access, accept it
+    - When prompted for access (by Windows Defender Firewall), click "Allow access"
 1. Install nvm
-    - `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
+    - Enter this command into your terminal: `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
 1. Move the 3 nvm lines from the bottom of `.bashrc` to the bottom of `.zshrc`
+    - This is the trickiest step! 
+    - Run this command to open your .bashrc config file: `code ~/.bashrc`
+    - Scroll down to the bottom of the file and cut the three lines at the bottom that look like this: 
+
+    ```j
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+        [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+    ```
+
+    - Run this command to open your .zshrc file: `code ~/.zshrc`
+    - Paste the nvm lines you cut from the .bashrc file down at the bottom of the .zshrc file
+1. Get rid of the distracting green highlighting in your terminal (optional, but recommended):
+    - In your .zshrc file (that you opened in the above step), find this line and uncomment it (i.e. delete the `#`):
+    ```
+    # DISABLE LS_COLORS="true"
+    ```
 1. Change the oh-my-zsh theme to 'bira'
+    - Scroll to the top of the .zshrc file and replace the `ZSH_THEME=` value with 'bira'
 1. Restart your terminal
-1. `nvm install --lts`
-
-1. Enable the automatic fixing of linting errors on file save by adding this to your `settings.json`
+1. Install the latest version of nvm 
+    - Run this command in your terminal: `nvm install --lts`
+1. In your Windows Terminal install some global npm packages
```

Comment:
This step is no longer needed. We're using [`eslint-config-eda`](https://www.npmjs.com/package/eslint-config-eda) now which no longer requires global npm installs.

Reply:
Thanks Don, I'll remove it!